invisible_cities/io/mcinfo_io.py

```diff
@@ -292,6 +291,65 @@ def compute_mchits_dict(mcevents:Mapping[int, Mapping[int, MCParticle]]) -> Mapp
     return mchits_dict
 
 
+def read_mchit_evt (mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
+                     event_number: int, last_row=0) -> ([tb.Table], [tb.Table], [tb.Table]):
```

Comment:
The return type of the function is just one tb.Table

Reply:
Ok!